app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/FeedbackUrlComposer.kt

```diff
@@ -16,27 +15,27 @@ class FeedbackUrlComposer(
      * Returns the feedback URL for the [session] if it can be composed
      * otherwise an empty string.
      *
-     * The [Frab schedule feedback URL][frabScheduleFeedbackUrl] is
+     * The [Frab schedule feedback URL][getFrabScheduleFeedbackUrl] is
      * composed from the session id.
      * For sessions extracted from the wiki of the Chaos Communication Congress
      * aka. "self organized sessions" an empty string is returned because
      * there is no feedback system for them.
      */
-    fun getFeedbackUrl(): String {
+    fun getFeedbackUrl(session: Session): String {
         if (session.originatesFromWiki) {
             return NO_URL
         }
         return if (session.originatesFromPretalx) {
             session.pretalxScheduleFeedbackUrl
         } else {
-            session.frabScheduleFeedbackUrl
+            getFrabScheduleFeedbackUrl(session.sessionId, frabScheduleFeedbackUrlFormatString)
         }
     }
 
-    private val Session.frabScheduleFeedbackUrl
-        get() =
-            if (frabScheduleFeedbackUrlFormatString.isEmpty()) NO_URL
-            else String.format(frabScheduleFeedbackUrlFormatString, sessionId)
+    private fun getFrabScheduleFeedbackUrl(sessionId: String, frabScheduleFeedbackUrlFormatString: String): String {
+        return if (frabScheduleFeedbackUrlFormatString.isEmpty()) NO_URL
+        else String.format(frabScheduleFeedbackUrlFormatString, sessionId)
```

Comment:
Maybe use braces to make it consistent looking with `getFeedbackUrl`.

Reply:
:heavy_check_mark: Agreed. Added in d6583a5dd41c1ab01cb82979b18e28a5d1e52546.